compiler/rustc_passes/src/check_attr.rs

```diff
@@ -376,6 +428,7 @@ impl<'tcx> CheckAttrVisitor<'tcx> {
 
     /// Checks that `#[optimize(..)]` is applied to a function/closure/method,
     /// or to an impl block or module.
+    // FIXME(#128488): this should probably be elevated to an error?
```

Comment:
I added this FIXME for https://github.com/rust-lang/rust/issues/128488, but elevating this to error may require a RFC tweak.